Tasks/Gradle/CodeAnalysis/gradlesonar.ts

```diff
@@ -0,0 +1,59 @@
+/// <reference path="../../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../../definitions/sonarqube-common.d.ts" />
+
+import path = require('path');
```

Comment:
Thanks for noticing this -- not exactly sure what I did, but I undid the last commit and it's fixed now.